bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -245,6 +245,162 @@ internal static unsafe class ElGamal
                 out ElGamalCiphertext.ElGamalCiphertextHandle handle);
         }
 
+        #endregion
+
+        #region ChaumPedersen
+
+        internal static unsafe class DisjunctiveChaumPedersenProof
```

Comment:
Long term might be smart to separate these into separate files. Okay as it stands. Another issue to encourage.